File: Problem-Set-1/sTaissa-cs50-problems-2020-x-mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h;

    do
    {
        h = get_int("Between 1 and 8, how tall is the pyramid?\n");
    }
    while (h < 1 || h > 8);

    for (int i = 1; i <= h; i++)
    {
        for (int p = h - i; p > 0; p--)
        {
            printf(" ");
        }
        for (int q = i; q > 0; q--)
        {
            printf("#");
        }

        printf("  ");

        for (int q = i; q > 0; q--)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

# input validation
while True:
    n = get_int("Height: ")
    if n <= 8 and n > 0: #between 1 and 8
        break

for row in range(n):
    for space in range(n-row, 1, -1): # make the first spaces
        print(" ", end="")
    for hash1 in range(row+1): # make the first pyramid
        print("#", end="")
    print("  ", end="") #make the gap between the pyramids
    for hash2 in range(row+1): #finishing the pirymid
        print("#", end="")
    print()